contracts/src/v0.8/dev/functions/FunctionsDON.sol

```diff
@@ -1,17 +1,17 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.6;
 
-import "../interfaces/OCR2DROracleInterface.sol";
+import "../interfaces/FunctionsDONInterface.sol";
 import "../ocr2/OCR2Base.sol";
 import "./AuthorizedOriginReceiver.sol";
 
 /**
- * @title OCR2DR oracle contract
+ * @title Functions Decentralized Oracle Network (DON) contract
  * @dev THIS CONTRACT HAS NOT GONE THROUGH ANY SECURITY REVIEW. DO NOT USE IN PROD.
  */
-contract OCR2DROracle is OCR2DROracleInterface, OCR2Base, AuthorizedOriginReceiver {
-  event OracleRequest(bytes32 indexed requestId, uint64 subscriptionId, bytes data);
-  event OracleResponse(bytes32 indexed requestId);
+contract FunctionsDON is FunctionsDONInterface, OCR2Base, AuthorizedOriginReceiver {
```

Comment:
Why call this DON? I think we use the name Oracle in other products even when they are backed by a full DON.

Reply:
Actually I agree with this.  Maybe I shoulda looked.  I figured we just did a drop-in replacement of OCR2DR for Functions.  I think that is all the name change we should have.

Reply:
> I think we use the name Oracle in other products even when they are backed by a full DON.
Do you know where? I just double checked and couldn't find one. The contract named `Oracle` was used in direct request, which makes sense since it was always just a single NOP.

Moving away from "Oracle" was originally brought up by Dev Rel in the initial review of the tooling repo and discussed further [here](https://www.notion.so/chainlink/Interfaces-touchpoints-naming-4b4eca1223824bb38184a3a51fd64e4e). If you disagree with the name, please raise concerns on that document.